packages/viz/src/lib/style/helpers/size-continuous-style.ts

```diff
@@ -143,3 +143,13 @@ function calculate(
     updateTriggers: { getRadius, getLineWidth }
```

Comment:
I think this updateTriggers won't be effective, as we don't specify a viewport-related variable, right?
cc/ @jesusbotella

Reply:
Yes, that update trigger will reevaluate styles when getRadius function reference changes (i.e. when updating styles definition).

If we want it to change, we would need to link it to a variable that changes when the points' size should change. I'll be careful linking triggers to viewport changes because reevaluating styles when panning the viewport without data change may not be ideal.

Reply:
In this particular case, I think the variable should be. **zoomLevel** indeed, not any other viewport change (panning should not trigger)

Reply:
`getRadius` is using `pixel2meters` function which is using layer to get the viewport and then, `metersPerPixel` which is finally used to calculate the size on the current zoom. Also, when the user changes the zoom, the function is called.

Do you think there is something wrong?